static/app/components/events/rrwebReplayer/index.tsx

```diff
@@ -39,7 +41,7 @@ const StyledRRWebReplayer = styled(RRWebReplayer)`
   .rr-controller {
     width: 100%;
     display: block;
-    padding: ${p => p.theme.space[2]}px 0;
+    padding: ${space(2)};
```

Comment:
The padding values are not equivalent here (`0` missing in the shorthand), is that intentional?